lib/action_cable/connection/identification.rb

```diff
@@ -35,7 +35,7 @@ def connection_identifier
         def connection_gid(ids)
           ids.map do |o|
             if o.respond_to? :to_global_id
-              o.to_global_id
+              o.to_global_id.to_s
```

Comment:
Maybe we should be using`to_gid_param` instead of `to_global_id.to_s` here, like we do in https://github.com/rails/actioncable/blob/master/lib/action_cable/channel/broadcasting.rb#L20